src/pages/privacy-policy.vue

```diff
@@ -103,51 +103,38 @@
       </p>
       <p>Examples of Cookies we use:</p>
       <list>
-        <list-item>
-          Session Cookies.&nbsp;We use Session Cookies to operate our Service.
-        </list-item>
+        <list-item>Session Cookies.&nbsp;We use Session Cookies to operate our Service.</list-item>
         <list-item>
           Preference Cookies.&nbsp;We use Preference Cookies to remember your preferences and
           various settings.
         </list-item>
-        <list-item>
-          Security Cookies.&nbsp;We use Security Cookies for security purposes.
-        </list-item>
+        <list-item>Security Cookies.&nbsp;We use Security Cookies for security purposes.</list-item>
       </list>
       <heading level="2">Use of Data</heading>
       <p>Vue.js Limited uses the collected data for various purposes:</p>
       <list>
-        <list-item>
-          To provide and maintain our Service
-        </list-item>
-        <list-item>
-          To notify you about changes to our Service
-        </list-item>
+        <list-item>To provide and maintain our Service</list-item>
+        <list-item>To notify you about changes to our Service</list-item>
         <list-item>
           To allow you to participate in interactive features of our Service when you choose to do
           so
         </list-item>
-        <list-item>
-          To provide customer support
-        </list-item>
-        <list-item>
-          To gather analysis or valuable information so that we can improve our Service
-        </list-item>
-        <list-item>
-          To monitor the usage of our Service
-        </list-item>
-        <list-item>
-          To detect, prevent and address technical issues
-        </list-item>
+        <list-item>To provide customer support</list-item>
```

Comment:
When in-lining prettier does some wacky stuff... It's pretty annoying. Could we revert these changes as it breaks the IDE syntax highlighting

Reply:
it does indeed :D will revert

Reply:
reverted